### PR TITLE
lcow: Added new pipeline

### PR DESCRIPTION
**Pull request by mbivolan**

### CODE REVIEW

File: scripts/lcow/BuildOpenGCS.ps1 [hunk 1]
```diff
@@ -0,0 +1,110 @@
+param (
+    [parameter(Mandatory=$true)]
+    [String] $ArtifactsPath,
+    [String] $DependeciesPath,
+    [String] $WorkDir,
+    [String] $LogDestination
+)
+
+$TEST_REPO = "https://github.com/Microsoft/opengcs.git"
```
Comment (ader1990) [inline scripts/lcow/BuildOpenGCS.ps1]: these params should all go to the jenkinsfile

File: scripts/lcow/BuildOpenGCS.ps1 [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+$TEST_BRANCH = "master"
+$TEST_DEPENCIES = @{
```
Comment (ader1990) [inline scripts/lcow/BuildOpenGCS.ps1]: typo

File: scripts/lcow/CommonFunctions.psm1 [hunk 3]
```diff
@@ -0,0 +1,105 @@
+function Download-File {
```
Comment (ader1990) [inline scripts/lcow/CommonFunctions.psm1]: add these functions to helpers.psm1

File: scripts/lcow/CopyArtifacts.ps1 [hunk 4]
```diff
@@ -0,0 +1,34 @@
+param (
+    [parameter(Mandatory=$true)]
+    [String] $KernelUrl,
+    [parameter(Mandatory=$true)]
+    [String] $InitrdUrl,
+    [parameter(Mandatory=$true)]
+    [String] $Destination
+)
+
+function Download-File {
```
Comment (ader1990) [inline scripts/lcow/CopyArtifacts.ps1]: reuse helpers

File: linux_pipeline/Jenkinsfile_container_images [hunk 5]
```diff
@@ -5,246 +5,136 @@ def PowerShellWrapper(psCmd) {
...
-                --kernel_version ${KERNEL_VERSION} \\
-                --clean_env ${CLEAN_ENV}
-            popd
-            '''
-        }
-      }
+def runLinuxKitTesting() {
+    dir ("lcow-linuxkit-${env.BUILD_NUMBER}-${env.BRANCH_NAME}") {
+        checkout scm
+        echo 'Staring linuxkit testing'
```
Comment (chvalean) [inline linux_pipeline/Jenkinsfile_container_images]: starting*

File: scripts/lcow/CommonFunctions.psm1 [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+    )
+    
+    if (Test-Path $BinariesPath) {
+        $env:PATH = $BinariesPath + ";" + $env:PATH
+    } else {
+        Write-Output "Error: Cannot find required binaries"
+    }
+    
+    $randomIdentifier = Get-Random -Maximum 1000
+    if (-not (Test-Path "C:\Docker-Workspace")){
```
Comment (ader1990) [inline scripts/lcow/CommonFunctions.psm1]: add this to a variable

File: scripts/lcow/CommonFunctions.psm1 [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+        New-Item -Type Directory -Path "C:\Docker-Workspace"
+    }
+    $dockerdDir = "C:\Docker-Workspace\DockerDir_${randomIdentifier}"
```
Comment (ader1990) [inline scripts/lcow/CommonFunctions.psm1]: same

File: scripts/lcow/Copy-Artifacts.ps1 [hunk 8]
```diff
@@ -0,0 +1,34 @@
+param (
+    [parameter(Mandatory=$true)]
+    [String] $KernelUrl,
+    [parameter(Mandatory=$true)]
+    [String] $InitrdUrl,
+    [parameter(Mandatory=$true)]
+    [String] $Destination
+)
+
+function Download-File {
```
Comment (ader1990) [inline scripts/lcow/Copy-Artifacts.ps1]: use a helpers function or move it there

Comment (ader1990) [reply scripts/lcow/Copy-Artifacts.ps1]: rename it to download

File: scripts/lcow/Prepare-Env.ps1 [hunk 9]
```diff
@@ -0,0 +1,83 @@
+param (
+    [String] $BinariesDestination,
+    [String] $ArtifactsPath,
+    [String] $ArtifactsDestination
+)
+
+$TEST_DEPENCIES = @{
```
Comment (ader1990) [inline scripts/lcow/Prepare-Env.ps1]: DEPENDENCIES

File: scripts/lcow/Prepare-Env.ps1 [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+        Copy-Item "$ArtifactsPath\initrd.img" "$Destination"
+    } elseif (Test-Path "$ArtifactsPath\lcow-initrd.img") {
+        Copy-Item "$ArtifactsPath\lcow-initrd.img" "$Destination\initrd.img"
+    } else {
+        Write-Output "Error: Cannot find initrd.img in folder: $ArtifactsPath"
+        exit 1
+    }
+}
+
+function Get-TestDependecies {
```
Comment (ader1990) [inline scripts/lcow/Prepare-Env.ps1]: DEPENDENCIES

File: scripts/lcow/Prepare-Env.ps1 [hunk 11]
```diff
@@ -0,0 +1,83 @@
...
+        [hashtable] $Binaries,
+        [String] $LocalDir,
+        [String] $Destination
+    )
+    
+    if (Test-Path $Destination) {
+        Remove-Item -Recurse $Destination
+    }
+    New-Item -Type Directory -Path $Destination
+    [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
```
Comment (ader1990) [inline scripts/lcow/Prepare-Env.ps1]: no need for this if you put it in download file

File: scripts/utils/powershell/helpers.psm1 [hunk 12]
```diff
@@ -4,6 +4,25 @@ Import-Module Dism
 
 $scriptPath = Split-Path -Parent $MyInvocation.MyCommand.Definition
 
+function Download-File {
```
Comment (ader1990) [inline scripts/utils/powershell/helpers.psm1]: add here the tlsv 12 enforcement

**Pull request merged**